Testing fails after installation according to README:

```
 ~/HBI/Projects/dabapush   13-large-jsonlndjson-files-create-oom-error  poetry shell
Spawning shell within /Users/muench/Library/Caches/pypoetry/virtualenvs/dabapush-y8hPjhdx-py3.10
 ~/HBI/Projects/dabapush   13-large-jsonlndjson-files-create-oom-error  . /Users/muench/Library/Caches/pypoetry/virtualenvs/dabapush-y8hPjhdx-py3.10/bin/activate
(dabapush-py3.10)  ~/HBI/Projects/dabapush   13-large-jsonlndjson-files-create-oom-error  dabapush create
2022-10-24 18:12:02.818 | DEBUG    | dabapush.Dabapush:__new__:40 - Staring DabaPush instance with gc: <dabapush.Configuration.Registry.Registry object at 0x118af1960> and cf: <dabapush.Configuration.ProjectConfiguration.ProjectConfiguration object at 0x118af2140>
2022-10-24 18:12:02.818 | DEBUG    | dabapush.main:cli:54 - Starting DabaPush in /Users/muench/HBI/Projects/dabapush from /Users/muench/HBI/Projects/dabapush
2022-10-24 18:12:02.819 | DEBUG    | dabapush.create_subcommand:create:30 - Creating project in /Users/muench/HBI/Projects/dabapush
project name: test
author name (split several authors with ";"): FVM
Should we configure readers and writers? [y/N]: N
2022-10-24 18:12:22.663 | DEBUG    | dabapush.Dabapush:pr_write:73 - writing the following project configuration: <dabapush.Configuration.ProjectConfiguration.ProjectConfiguration object at 0x118af2140>
(dabapush-py3.10)  ~/HBI/Projects/dabapush   13-large-jsonlndjson-files-create-oom-error  pytest
=================================================================================================================================================================================== test session starts ====================================================================================================================================================================================
platform darwin -- Python 3.10.8, pytest-7.1.3, pluggy-1.0.0
rootdir: /Users/muench/HBI/Projects/dabapush, configfile: pyproject.toml
plugins: cov-4.0.0
collected 45 items                                                                                                                                                                                                                                                                                                                                                                         

tests/test_Dabapush.py ..sss....sssssss                                                                                                                                                                                                                                                                                                                                              [ 35%]
tests/test_package.py F                                                                                                                                                                                                                                                                                                                                                              [ 37%]
tests/Configuration/test_Configuration.py ssF.sssssFs                                                                                                                                                                                                                                                                                                                                [ 62%]
tests/Configuration/test_FileWriterConfiguration.py F                                                                                                                                                                                                                                                                                                                                [ 64%]
tests/Configuration/test_Registry.py ss..F....                                                                                                                                                                                                                                                                                                                                       [ 84%]
tests/Reader/test_TwacapicReader.py ss.ss                                                                                                                                                                                                                                                                                                                                            [ 95%]
tests/Writer/test_Writer.py ..                                                                                                                                                                                                                                                                                                                                                       [100%]/Users/muench/Library/Caches/pypoetry/virtualenvs/dabapush-y8hPjhdx-py3.10/lib/python3.10/site-packages/coverage/report.py:81: CoverageWarning: Couldn't parse Python file '/Users/muench/HBI/Projects/dabapush/dabapush/Pipeline.py' (couldnt-parse)
  coverage._warn(msg, slug="couldnt-parse")


========================================================================================================================================================================================= FAILURES =========================================================================================================================================================================================
_______________________________________________________________________________________________________________________________________________________________________________________ test_version _______________________________________________________________________________________________________________________________________________________________________________________

    def test_version():
>       assert __version__ == "0.1.0"
E       AssertionError: assert '0.2.0' == '0.1.0'
E         - 0.1.0
E         ?   ^
E         + 0.2.0
E         ?   ^

tests/test_package.py:5: AssertionError
_____________________________________________________________________________________________________________________________________________________________________________________ test_get_reader ______________________________________________________________________________________________________________________________________________________________________________________

conf = <dabapush.Configuration.Registry.Registry object at 0x10db38190>

    def test_get_reader(conf: Registry):
>       assert conf.get_reader("twacapic") is not None
E       AssertionError: assert None is not None
E        +  where None = <function Registry.get_reader at 0x10db16320>('twacapic')
E        +    where <function Registry.get_reader at 0x10db16320> = <dabapush.Configuration.Registry.Registry object at 0x10db38190>.get_reader

tests/Configuration/test_Configuration.py:23: AssertionError
_____________________________________________________________________________________________________________________________________________________________________________________ test_get_writer ______________________________________________________________________________________________________________________________________________________________________________________

conf = <dabapush.Configuration.Registry.Registry object at 0x10db92a10>

    def test_get_writer(conf: Registry):
>       assert conf.get_writer("twacapic") is not None
E       AssertionError: assert None is not None
E        +  where None = <function Registry.get_writer at 0x10db163b0>('twacapic')
E        +    where <function Registry.get_writer at 0x10db163b0> = <dabapush.Configuration.Registry.Registry object at 0x10db92a10>.get_writer

tests/Configuration/test_Configuration.py:63: AssertionError
___________________________________________________________________________________________________________________________________________________________________________________ test_make_file_name ____________________________________________________________________________________________________________________________________________________________________________________

writer_config = <dabapush.Configuration.FileWriterConfiguration.FileWriterConfiguration object at 0x10db3b700>

    def test_make_file_name(writer_config: FileWriterConfiguration):
        regex = re.compile(r"(\d{2}-){3}\d{4}-test")
        writer_config.set_name_template("$name")
>       name = writer_config.make_file_name()

tests/Configuration/test_FileWriterConfiguration.py:15: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <dabapush.Configuration.FileWriterConfiguration.FileWriterConfiguration object at 0x10db3b700>, additional_keys = {}

    def make_file_name(self, additional_keys: dict = {}) -> str:
        """
    
        Parameters
        ----------
        additional_keys :
            dict:  (Default value = {})
        additional_keys :
            dict:  (Default value = {})
        additional_keys: dict :
             (Default value = {})
    
        Returns
        -------
    
        """
        now = datetime.now()
        return Template(self.name_template).substitute(
            **{
                "date": datetime.strftime(now, "%Y-%m-%d"),
                "time": datetime.strftime(now, "%H%M"),
                "name": self.name,
                "id": self.id,
>               "type": self.type,
                **additional_keys,
            }
        )
E       AttributeError: 'FileWriterConfiguration' object has no attribute 'type'

dabapush/Configuration/FileWriterConfiguration.py:46: AttributeError
_____________________________________________________________________________________________________________________________________________________________________________________ test_list_reader _____________________________________________________________________________________________________________________________________________________________________________________

conf_loaded_single_reader = <dabapush.Configuration.ProjectConfiguration.ProjectConfiguration object at 0x10db908e0>

    def test_list_reader(conf_loaded_single_reader: ProjectConfiguration):
        entries = conf_loaded_single_reader.list_readers()
>       assert entries[0].name == "name"
E       AssertionError: assert 'default' == 'name'
E         - name
E         + default

tests/Configuration/test_Registry.py:56: AssertionError
---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- Captured stderr setup -----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------
2022-10-24 18:12:41.877 | DEBUG    | dabapush.Configuration.Registry:get_reader:66 - Creating Configuration instance from description, moduleName, className.
2022-10-24 18:12:41.877 | DEBUG    | dabapush.Configuration.ProjectConfiguration:add_reader:69 - Currently configured readers: default,reader1,name

---------- coverage: platform darwin, python 3.10.8-final-0 ----------
Coverage HTML written to dir tests/coverage

================================================================================================================================================================================= short test summary info ==================================================================================================================================================================================
FAILED tests/test_package.py::test_version - AssertionError: assert '0.2.0' == '0.1.0'
FAILED tests/Configuration/test_Configuration.py::test_get_reader - AssertionError: assert None is not None
FAILED tests/Configuration/test_Configuration.py::test_get_writer - AssertionError: assert None is not None
FAILED tests/Configuration/test_FileWriterConfiguration.py::test_make_file_name - AttributeError: 'FileWriterConfiguration' object has no attribute 'type'
FAILED tests/Configuration/test_Registry.py::test_list_reader - AssertionError: assert 'default' == 'name'
========================================================================================================================================================================= 5 failed, 16 passed, 24 skipped in 1.94s =========================================================================================================================================================================
```

--- a/dabapush/Reader/NDJSONReader.py
+++ b/dabapush/Reader/NDJSONReader.py
@@ -12,7 +12,7 @@ from .Reader import Reader
 class NDJSONReader(Reader):
     """Reader to read ready to read NDJSON data.
     It matches files in the path-tree against the pattern and reads all
-    files and all lines in these file as JSON.
+    files and all lines in these files as JSON.
 
     Attributes
     ----------

